pyproject.toml

```
@@ -4,12 +4,13 @@ build-backend = "setuptools.build_meta"
 
 [project]
 name = "lm-buddy"
-version = "0.4.0"
+version = "0.4.1"
```

```suggestion
version = "0.5.0"
```

Since its a new feature, not a patch.